system/ee/EllisLab/ExpressionEngine/Controller/Utilities/MemberImport.php

```diff
@@ -116,9 +116,14 @@ public function index()
 						'date_format' => array(
 							'type' => 'radio',
 							'choices' => array(
-								'%n/%j/%Y' => 'mm/dd/yyyy',
-								'%j/%n/%Y' => 'dd/mm/yyyy',
-								'%j-%n-%Y' => 'dd-mm-yyyy',
+								'%n/%j/%Y' => 'm/d/yyyy',
+								'%m/%d/%Y' => 'mm/dd/yyyy',
+								'%j/%n/%Y' => 'd/m/yyyy',
+								'%d/%m/%Y' => 'dd/mm/yyyy',
+								'%j-%n-%Y' => 'd-m-yyyy',
+								'%d-%m-%Y' => 'dd-mm-yyyy',
+								'%j.%n.%Y' => 'd.m.yyyy',
+								'%d.%m.%Y' => 'dd.mm.yyyy',
```

Comment:
Please look on the original code:

```
'%n/%j/%Y' => 'mm/dd/yyyy',
'%j/%n/%Y' => 'dd/mm/yyyy',
'%j-%n-%Y' => 'dd-mm-yyyy',
```

In fact mm/dd/yyyy produce date without zero for days and months. The correct code should be:

`
'%m/%d/%Y' => 'mm/dd/yyyy',
`
or
`
'%n/%j/%Y' => 'm/d/yyyy',
`

So I included both variants for each format.

Reply:
This only adds the options to the Member Import screen. Why only add them here?

Reply:
Can you outline what you've done to test? Formats like `m/d/yyyy` will not work, because the month and day is ambiguous, and PHP will get the date wrong frequently.